frontend/src/components/SearchBar.tsx

```diff
@@ -136,7 +139,7 @@ const useStyles = makeStyles((theme) => ({
     top: '50%',
     transform: 'translateY(-50%)',
     fontSize: '1.5rem',
-    color: theme.palette.grey[300]
+    color: '#71767A'
```

Comment:
Do we need to change theme.palette.grey instead? or is it ok to hardcode this color in here?

Reply:
I noticed it was hardcoded like this in other components, but I can look into changing the mui palette.